components/prospects/ProspectCard.tsx

```diff
@@ -0,0 +1,23 @@
+import { Card, CardActionArea, CardContent } from '@mui/material'
```

Comment:
What's up with this file? Why did you create another one?

Reply:
I asked Petru some days ago about my task and he said it's not worth it to make a single component because it would be too hard.